### ISSUE TITLE
Status tracking bar extending past frame

### ISSUE THREAD

**Issue by Xorag** [Nevcairiel/Bartender4#163]
Hello, after the recent patch I've noticed on the current and previous version of Bartender4 that the status bar expands past the edge of the frame like so:

![image](https://user-images.githubusercontent.com/42698965/236656945-bb59219a-d98d-4eb5-a6dd-b6a7af0c058d.png)

I can't really provide more info other than it happened once 10.1 released.

**Comment by AbelardTullus** [comment]
I'm having this issue as well, haven't come across a fix as of yet.
![image](https://github.com/Nevcairiel/Bartender4/assets/20971561/be054942-cb9e-44fa-b37f-8b194b29acc3)

**Comment by Cuphat** [comment]
Yeah, width of the bar doesn't work properly and the actual width is unchanging, so it can either be bigger or smaller than the width that is set for the status tracking bar.
![image](https://github.com/Nevcairiel/Bartender4/assets/15843710/5b034764-41dd-44a6-9383-af06653d8f79)

**Comment by Malivil** [comment]
Ok, different question: Anyone have a replacement for Bartender since this seems to be abandoned (or in the process at the very least)?

**Comment by Ketrel** [comment]
This is still happening in TWW
